content/documentation/3-stream-developer-guides/3-programming-models/3-kafka-streams.md

```diff
@@ -6,25 +6,125 @@ description: 'Kafka Streams Processing Developer Guide'
 
 # Kafka Streams Processing
```

Comment:
@sobychacko is looking to add programming model guides for SI, Flux, KStreams, and Functions [see: https://github.com/spring-cloud/spring-cloud-dataflow/issues/3095]. The goal was to hopefully showcase the business logic in the 4 models. Perhaps we can remove this section from the commit until we decide what business logic we want to showcase.